client/src/test/java/TestRunner.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+ClientDesktop='../../../build/libs/com.fiskaly.kassensichv.client.desktop-0.0.1-alpha.jar'
+
+#run GeneralSMATest
+javac -cp ${ClientDesktop} GeneralSMATest.java
+
+java -cp ${ClientDesktop} GeneralSMATest.java
```

Comment:
Remove the `.java` suffix

Reply:
Removed the suffix. bd854cb27f34dd37414900677d5b6cb7960dba01